client/src/components/ClickableBox.tsx

```diff
@@ -34,14 +34,14 @@ export const ClickableBox = ({
 
     return (
         <View style={{marginVertical: 10, marginHorizontal: 10}}>
-            <RelativeView>
+            <RelativeView style={{backgroundColor: '#f2f2f2'}}>
                 {hasIcon &&
                     <>
                         <TextToSpeechIcon text={name}
                                           gender={userGender as Gender}
                                           style={{position: 'absolute', top: 3, left: 6, zIndex: 1}}/>
```

Comment:
Add backgroundColor: '#f2f2f2' to TextToSpeechIcon too
Also add backgroundColor to FavoriteIcon (look at the comment in FavoriteIcon where you should add it)

Reply:
done